README.md

```diff
@@ -1,2 +1,87 @@
+    - Scope (e.g. excludes any changes to modular kit or API, independent of Python 3 project)
+    - Deliverables (e.g. Board, Software, Documentation for students and volunteers)
+    - Acceptance Criteria (e.g. is demonstrated to run a defined piece of software on a test robot without fault, with a certain response time for libkoki markers, etc.)
+- The _Dev Team_ may also define some limits, such as:
+    - Cut-off date for deployment at SR2020
+    - Cost per unit limit
+    - Evidence shown that more than one board has been considered
+- The _Dev Team_ may consult the _Core Team_ or Trustees to assist them with requirements. For example, in the case of a Project that would have a high implementation cost, the _Dev Team_ would need to communicate the feasibility of it with the Trustees.
+- This specification is presented to and ratified by the _Dev Team_ (either on the _Dev Team_ mailing list or at a _Dev Team_ meeting). Any sensitive information, such as a budget, will need to be privately discussed by the _Dev Team_ and _project group_.
+- Development work happens.
+- If changes to the specification are needed, these can be discussed with the _Dev Team_.
+- Development reaches a point where the project group is happy the work is done and is shippable.
+- _Dev Team_ reviews the equipment against the agreed specification and decides whether to recommend it to the _Core Team_.
+- The process is now passed onto the _Core Team_ who are responsible for deciding when and how to deploy it (e.g for SR 2020).
+    - If it meets the needs of the competition, it is the _Core Team_’s responsibility to ensure new kit is purchased, tested and deployed (although it’s likely to be the _project group_ who do this).
+    - If it doesn’t, the _Core Team_ must decide whether to proceed anyway at risk, or to continue with the existing kit and pause expansion.
+
+### Project of minor impact: Addition to the API (e.g. making an “R.ruggeduinos[0].PWM()” )
+
+As this is notionally a simpler thing, but still in scope as it impacts the competition cycle, the process can be simpler.
+
+- Developer wants to add the feature and is pointed in the direction of the appropriate _project group_ by the _Dev Team, or creates one if it does not exist.
+- The _project group_ determines that there is no need for a specification due to minor impact.
+    - A change of minor impact does not have significant cost or change to the working of the competition.
+    - The _Dev Team_ may determine that a project is of minor impact and require a specification.
+- Developer updates/opens a new ticket on the appropriate GitHub Project, adding to the relevant repo also if one exists.
+- The _project group_ will define testing and documentation requirements.
+- Developer submits a pull request. 
+- The _project group_ checks it meets the requirements and approves or rejects.
+- The _Dev Team_ will review the new version and decides whether to recommend it to the _Core Team_.
```

Comment:
I'm concerned by the ordering here -- if at this point the Dev Team decide not to proposed the change to the Core Team, does that mean that the change has to be backed out of the repo (I'm not clear at what point we merge the changes to the repo)?
What happens if the Core Team for some reason don't want the change? (I'd hope this was rare)

Reply:
> I'm concerned by the ordering here -- if at this point the Dev Team decide not to proposed the change to the Core Team, does that mean that the change has to be backed out of the repo 

The Dev Team would work with the project group to make the appropriate changes for it to be shippable. In some cases this would mean reverting things. For example if the Project Group decides that the kit really badly needs support for flying pink unicorns, but also breaks vision.

>(I'm not clear at what point we merge the changes to the repo)?

The Project Group merges it when they approve. The Dev Team recommends a specific revision, i.e commit hash or release number to the Core Team

> What happens if the Core Team for some reason don't want the change? (I'd hope this was rare)

I would hope that the working together nature of the Dev Team would prevent this. If it did occur though, the Core Team should stick to the latest stable version that they are happy with.